avm/res/db-for-my-sql/flexible-server/tests/e2e/max/dependencies2.bicep

```diff
@@ -31,15 +31,16 @@ resource keyVault 'Microsoft.KeyVault/vaults@2023-02-01' = {
     }
     tenantId: tenant().tenantId
     enablePurgeProtection: true // Required for encryption to work
-    softDeleteRetentionInDays: 7
+    softDeleteRetentionInDays: 90
     enabledForTemplateDeployment: true
     enabledForDiskEncryption: true
     enabledForDeployment: true
     enableRbacAuthorization: true
+    enableSoftDelete: true
```

Comment:
as far as I know, this is the dafault and actually you cannot set it to false even if you wanted to. In other words, this property should be redundant.

Reply:
Fixed.